### DISCUSSION TITLE
一个小细节问题

### DISCUSSION THREAD

**Discussion by zkzmba793** [the1812/Bilibili-Evolved#1440] [功能建议 / Ideas]
脚本版本1.11.11

在视频页面种用 快捷键调节音量 和 滚轮上下滑调节音量 显示的样式不一样
使用 滚轮 时：
![1](https://user-images.githubusercontent.com/73212412/104026449-e297aa00-5200-11eb-867b-2fb2b1bf6202.png)

使用 快捷键 时：
![2](https://user-images.githubusercontent.com/73212412/104026520-00fda580-5201-11eb-83b6-6361ff865845.png)

希望滚轮的调节音量时的样式能统一成快捷键调节音量的样式

**Comment by the1812** [comment]
滚轮的是 b 站原版的...

**Reply by zkzmba793** [reply]
> 滚轮的是 b 站原版的...

我知道啊，但是我觉得改成 快捷键 的样式 看起来比较舒服一点，而且你倍速快捷键也是用的这个，干脆就一起统一吧233

**Reply by the1812** [reply]
最多就把颜色换过来, 图标还是有些细微的差别的

**Reply by zkzmba793** [reply]
> 最多就把颜色换过来, 图标还是有些细微的差别的

那你就改一下颜色也行

**Comment by Geeyun-JY3** [comment]
是怎么做到用滚轮上下滑调节音量的……我这边快捷键出来的还是第一种样式……

**Reply by the1812** [reply]
网页全屏或全屏状态下才有

**Reply by Geeyun-JY3** [reply]
对啊我网页全屏和全屏都是第一种

**Reply by Geeyun-JY3** [reply]
哦你说的是滚轮调节网页全屏或全屏状态下才有啊……看来我这边是有脚本或插件和原生功能冲突了

**Reply by Geeyun-JY3** [reply]
搞清楚了，是别的脚本导致我快捷键调节音量显示的样式是原生样式

**Reply by Geeyun-JY3** [reply]
> 我刚刚看了下我用的预览离线版的，预览版的不知道为啥没有（可能忘了？）

最新预览离线版在网页全屏或全屏状态下滚轮好像确实失效了？bug？@the1812

**Reply by Geeyun-JY3** [reply]
> > 搞清楚了，是别的脚本导致我快捷键调节音量显示的样式是原生样式
> 
> 不是吧？我又试了一次，预览版没有那个，离线的就有

我的意思是我一开始说用快捷键调节音量出来的是第一种样式。和本脚本无关。
和下面说的滚轮不是一个事。

**Reply by zkzmba793** [reply]
> > > 搞清楚了，是别的脚本导致我快捷键调节音量显示的样式是原生样式
> > 
> > 
> > 不是吧？我又试了一次，预览版没有那个，离线的就有
> 
> 我的意思是我一开始说用快捷键调节音量出来的是第一种样式。和本脚本无关。
> 和下面说的滚轮不是一个事。

我上面说的太乱了.... 全删了，理清一下吧，我去chrome无痕浏览全部扩展关掉只开这个脚本，确实是预览版去试 快捷键 的就没有第二种样式，预览离线的就没有....  有点奇怪，可能只是我是个例

然后滚轮的话我是能用的，我在上面说的（滚轮失效问题，评论我删了）只是偶尔会出现失效的问题，可能只是我自己装的扩展或别的脚本的问题

**Comment by Geeyun-JY3** [comment]
顺便也扣个细节吧：当倍速 <1.0 时，这个图标换个反方向的吧
![image](https://user-images.githubusercontent.com/49396421/104049896-b55af400-5220-11eb-8ecd-7db4d91a15c0.png)

**Reply by Geeyun-JY3** [reply]
或者可以参考 YouTube 的做法，将这个提示分为两个，一个指示当前倍数，一个指示倍数变化
![image](https://user-images.githubusercontent.com/49396421/104051264-d7ee0c80-5222-11eb-8191-130a2c56cab2.png)
![image](https://user-images.githubusercontent.com/49396421/104051361-fbb15280-5222-11eb-8c51-25047de72d51.png)

**Reply by CKylinMC** [reply]
我觉得反方向会有一种视频倒放的错觉

**Reply by Geeyun-JY3** [reply]
> 我觉得反方向会有一种视频倒放的错觉

反正 YouTube 是这么做的🌚我是觉得倍速 <1.0 时现在这图标就有种告诉你在快进的意思

**Reply by Geeyun-JY3** [reply]
或者去 MDI 找找有没有这个意思的图标🙈

**Reply by the1812** [reply]
虽然倍速慢了但是进度条还是向右走的吧

**Reply by Geeyun-JY3** [reply]
两边都说得通🌚我感觉 YouTube 这种也挺好。还是这图标的意思不太明确。

**Reply by CKylinMC** [reply]
我直接搜索slow play的图标给的提示都是step-play的：

![差不多长这样](https://user-images.githubusercontent.com/11648014/104085906-f76e4f00-528d-11eb-8c7c-d01b37e47098.png)

然后我去看mdi的图标表之后就在想：

为什么倍速播放一定要带个箭头呢？本质上不是速度在变化吗？（开玩笑）

![速度表](https://user-images.githubusercontent.com/11648014/104085855-89298c80-528d-11eb-8c46-0bf87f66e557.png)

**Comment by BakaJzon** [comment]
可以发送滚轮事件嘛？
